runtime/src/main/java/org/corfudb/runtime/view/ReadOptions.java

```diff
@@ -0,0 +1,25 @@
+package org.corfudb.runtime.view;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Client read options.
+ *
+ * <p>Created by Maithem on 7/9/19.</p>
+ */
+
+@Builder
+@Data
+public class ReadOptions {
+    final boolean ignoreTrim;
```

Comment:
Please add comments for all these fields.

Reply:
done

Reply:
Comments

Reply:
+1

Reply:
done